Readme.md

```diff
@@ -8,9 +8,28 @@
 
 In v16.1 and higher, you can use the [Template Gallery](https://docs.devexpress.com/WindowsForms/16492/whats-installed/template-gallery) to create a [GridView](https://docs.devexpress.com/WindowsForms/DevExpress.XtraGrid.Views.Grid.GridView) descendant with all required infrastructure classes. Use the `Custom Data Grid` template for this purpose.
 
-The current example shows how to create a GridView descendant without using this template. This descendant allows users to delete records when they press the `Delete` key. Build the project prior to opening the `Form1` form in the Visual Studio designer. 
-  
-See the following article for more information: [Create GridView descendants](https://www.devexpress.com/Support/Center/p/A859).
+The current example shows how to create a `GridView` descendant without using this template. This descendant allows users to delete records when they press the `Delete` key. Build the project prior to opening the `Form1` form in the Visual Studio designer. 
+
+## View Descendant Concepts
+This section describes basic concepts to create a descendant of the `GridView`, demonstrated in the attached project.
+
+You typically need to create descendants of the following classes to implement custom functionality for a `GridView`:
+
+- `GridView` - The component that exposes customization settings. References all infrastructure classes.
+- `GridControl` - The control that you place on the form. This is a container for your View descendant.
+- `GridInfoRegistrator` - Contains the View's registration information.
+- `GridHandler` - Processes mouse and keyboard actions.
+- `GridViewInfo` - Calculates display information used to draw the View's elements.
+- `GridViewPrintInfo` - Calculates the information necessary to print the View.
+
+Override the `ViewName` property for the `GridView` descendant. This property should return a unique View name.
+
+Do the following to register the `GridView` descendant for design-time use:
+
+- Create a `GridInfoRegistrator` class descendant. This class's `ViewName` property must return the same value as your View's `ViewName` property.
+- Override the `RegisterAvailableViewsCore` method in your `GridControl` descendant.
+- If you want your `GridControl` to create an instance of your `GridView` descendant on the control's initialization, override the `GridControl.CreateDefaultView` method.
```

Comment:
Is it necessary to say "your"? For example, can we change both instances of "your" to "the"?

`If you want the `GridControl` to create an instance of the `GridView` descendant on the control's initialization, override the `GridControl.CreateDefaultView` method.`

Reply:
```suggestion
- If you want the custom `GridControl` to create an instance of the `GridView` descendant on the control's initialization, override the `GridControl.CreateDefaultView` method.
```